hnn/specfn.py

```diff
@@ -0,0 +1,222 @@
+# specfn.py - Average time-frequency energy representation using Morlet
+# wavelet method
+#
+# v 1.10.2-py35
+# rev 2017-02-21 (SL: fixed an issue with indexing)
+# last major: (SL: more comments on the units of Morlet Spec)
+# 11-29-2020: BC removed code that no longer uses in preparation for
+# hnn-core integration
+
+import numpy as np
+import scipy.signal as sps
+
+
+# MorletSpec class based on a time vec tvec and a time series vec tsvec
+class MorletSpec():
```

Comment:
There is now a spectrogram function in hnn-core. I think you might be able to get rid of this file too

Reply:
https://jonescompneurolab.github.io/hnn-core/stable/generated/hnn_core.viz.plot_tfr_morlet.html#hnn_core.viz.plot_tfr_morlet

Reply:
That'll be a nice improvement. Won't address in this PR